src/ui/component/textHighlighter/textHighlighter.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FC } from 'react'
+import { splitTextByTerm } from '@/util/splitTextByTerm/splitTextByTerm'
+
+type TextHighlighterProps = {
+  text: string
+  termToHighlight: string
+}
+export const TextHighlighter: FC<TextHighlighterProps> = ({
+  text,
+  termToHighlight: textToHighlight
```

Comment:
I'm not sure to understand why it's renamed here, for me `termToHighlight` was fine but I think that if you prefer `textToHighlight` you could use this name dirctly, no?

Reply:
Good catch thanks! The component prop type was renamed but I forgot to rename the prop accordingly.

Reply:
Ideally, a comparison function should be provided as part of the props, so that you can "scan" the term to highlight in the text. But the implementation is a bit tricky, from what I can see.